<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>串口配置</string>
    </property>
    <widget class="QComboBox" name="portbox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="frequencybox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="databox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="stopBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="binaryBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>null</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="serialButton1">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>260</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>端口号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>传输速率</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>结束位</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>校验</string>
     </property>
    </widget>
    <widget class="QPushButton" name="serialButton2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>断开串口</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>901</width>
      <height>571</height>
     </rect>
    </property>
    <property name="title">
     <string>模块测试</string>
    </property>
    <widget class="QPushButton" name="mButton1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>281</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string>测试模块1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mButton2">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>50</y>
       <width>281</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string>测试模块2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mButton3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>281</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string>测试模块3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mButton4">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>310</y>
       <width>281</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string>测试模块4</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>数据库配置</string>
    </property>
    <widget class="QPushButton" name="DbButton1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>连接数据库</string>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>241</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="DbButton2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>断开数据库</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1212</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>类型</string>
    </property>
    <addaction name="action4438"/>
    <addaction name="action7139"/>
    <addaction name="action1278"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <addaction name="menu1"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action4438">
   <property name="text">
    <string>4438</string>
   </property>
  </action>
  <action name="action7139">
   <property name="text">
    <string>7139</string>
   </property>
  </action>
  <action name="action1278">
   <property name="text">
    <string>1278</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
